src/common/test/summarize-profile.js

```diff
@@ -24,18 +24,19 @@ describe('summarize-profile', function () {
 
     let i = 0;
     for (const { category, samples, percentage } of [
-      { category: 'idle', samples: 142, percentage: 0.48464163822525597 },
+      { category: 'wait', samples: 141, percentage: 0.4812286689419795 },
```

Comment:
Curious your thoughts on how this test is working so far, especially considering it's a raw dump of a profile, as opposed to a smaller carefully constructed profile.